adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -97,7 +101,7 @@
     }
 
     AuthenticationResult(String accessToken, String refreshToken, Date expires, boolean isBroad,
-            UserInfo userInfo, String tenantId, String idToken) {
+                         UserInfo userInfo, String tenantId, String idToken, Date extendedExpires) {
```

Comment:
This is the default format setting for Java in Android Studio. Any idea how to fix this if this is not correct? Thank you.